openapi/openapi.yaml

```diff
@@ -452,6 +452,20 @@ paths:
         - namespaces
         - v3
 
+  '/namespaces/{ns_name}/imports/collections':
```

Comment:
This seems like it should be a under /collection-imports (`GET /collection-imports/?namespace={ns_name}` or `GET /collection-imports/namespace/{ns_name}` and implemented as part of collection-imports view.

Ditto for `GET /namespaces/{ns_name}/collections/`. Seems like it should be `GET /collections/{ns_name}/` so it could be a filtered view in views/collections.py